Gopkg.toml

```diff
@@ -9,7 +9,7 @@ source = "gopkg.in/fsnotify/fsnotify.v1"
 
 [prune]
   go-tests = true
-  #unused-packages = true
+  unused-packages = true
```

Comment:
Thanks @shashidharatd , one question, how about go through Gopkg.toml and check if we can remove all the unused `[[override]]`?

Reply:
AFAIK, these override directive will not cause inclusion of unused packages. they are used to override the package version or source etc...

Reply:
Anyways, i will update the packages based on what is generated by kubebuilde v1.0.4